ArjunaJTS/jtax/tests/classes/com/hp/mwtests/ts/jta/jts/tools/JTSObjStoreBrowserTest.java

```diff
@@ -230,11 +230,11 @@ public void jtsMBeanTest() throws Exception {
         } catch (HeuristicHazard e) {
         }
 
-        createObjStoreBrowser(true); // start an mbean server and object store browser
+		ObjStoreBrowser osb = createObjStoreBrowser(true); // start an mbean server and object store browser
```

Comment:
You replaces spaces with tabs resulting in the code no longer lining uo

Reply:
+1, strange IDE behaviour, probably because part of the file is tab indented. I will fix it.